content/en/docs/pipelines/installation/overview.md

```diff
@@ -121,8 +142,7 @@ from GCP Marketplace. You can deploy Kubeflow Pipelines to an existing or new
 GKE cluster and manage your cluster within GCP.
 
 Installation guide
-: [Deploy Kubeflow Pipelines from Google Cloud
-  Marketplace](https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/guide.md)
+: [AI Platform Pipelines documentation](https://cloud.google.com/ai-platform/pipelines/docs)
```

Comment:
I think it has to be explicitly mentioned at least in the header:

> "## AI Platform Pipelines"

that both the AI Platform and the AI Platform Pipelines are _Google Cloud products_, since the Kubeflow docs service many platforms and on-prem solutions. This can be otherwise quite confusing to users who have never used AIP (I have, so I'm aware of this).

I suggest to change the subheading to:

> "## Google Cloud AI Platform Pipelines"